README.md

```diff
@@ -1,10 +1,18 @@
-# SernHandler
+<div align="center">
+  <img src="https://raw.githubusercontent.com/sern-handler/.github/main/sern-rounded.png" width="900px">
```

Comment:
Is there anyway u could make it a little less pink possibly ( the background ) ? the color of logo and the background dont contrast and it makes the handler logo blend in